lib/rubocop/ast/node/arg_node.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module AST
+    # A node extension for `arg`, `optarg`, `restarg`, `kwarg`, `kwoptarg`,
+    # `kwrestarg` and `blockarg` nodes. This will be used in place of a plain
+    # node when the builder constructs the AST, making its methods available
+    # to all `arg` nodes within RuboCop.
+    class ArgNode < Node
+      # Returns the name of an argument.
+      #
+      # @return [Symbol] the name of the argument
+      def name
```

Comment:
it can return `nil` for `def m(*); end` and `def m(**); end`

Reply:
and `...`

Reply:
These will just return nil already but I’m adding tests for them.